### DISCUSSION TITLE
Discuss "Release notes"

### DISCUSSION THREAD

**Discussion by cwen0** [chaos-mesh/chaos-mesh#3432] [General]
As introduced in https://github.com/chaos-mesh/chaos-mesh/wiki/Release-Guide, we draft release notes based on `CHANGELOG.md`, but the release notes lack some infromation about the updates of the dependent components, such as in [release v2.3.0-alpha.3](https://github.com/chaos-mesh/chaos-mesh/releases/tag/v2.3.0-alpha.3), we update chaos-tproxy versiobn, but we can't get any useful information from this release notes. 

* Bump chaos-tproxy to v0.5.1 https://github.com/chaos-mesh/chaos-mesh/pull/3412 

I suggestion we should add introduction about the new version of dependent components, like this: 

* Bump chaos-tproxy to v0.5.1 https://github.com/chaos-mesh/chaos-mesh/pull/3412 
     * Fix xxxx 
     * xxxx  

If the dependent compenents have the completed release notes, we also can add the links of release notes.

**Comment by g1eny0ung** [comment]
> I suggestion we should add introduction about the new version of dependent components, like this: 
>
> * Bump chaos-tproxy to v0.5.1 https://github.com/chaos-mesh/chaos-mesh/pull/3412
     * Fix xxxx 
     * xxxx  

+1. It benefits everybody. We can get more information directly from the release notes instead of clicking on the link to see it.

It makes sense to make that extra bit of work.
